jib-gradle-plugin/CHANGELOG.md

```diff
@@ -7,6 +7,18 @@ All notable changes to this project will be documented in this file.
 
 - Also tries `.exe` file extension for credential helpers on Windows. ([#2527](https://github.com/GoogleContainerTools/jib/issues/2527))
 - New system property `jib.skipExistingImages` (false by default) to skip pushing images (manifests) if the image already exists in the registry. ([#2360](https://github.com/GoogleContainerTools/jib/issues/2360)
+- _Incubating feature_: can now configure desired platform (architecture and OS) to select the matching manifest from a Docker manifest list or an OCI image index for a base image. Currently supports building only one image. ([#1567](https://github.com/GoogleContainerTools/jib/issues/1567))
```

Comment:
Have we built the feature to support an OCI image index yet?

Reply:
Ah, that's a good point. Will fix.